src/components/mainpage/chatscreen/components.mainpage.chatscreen.chatroom.css

```
@@ -0,0 +1,50 @@
+.chatroom {
+  display: flex;
+  flex-direction: column;
+  overflow: scroll;
```

처음 화면에서 스크롤바가 이상한 위치에 나타나는데 이렇게 수정하시면 해결됩니다...!

```suggestion
  overflow: auto;
```